### PR TITLE
Add an explanation for blueprint appliances status

### PR DESCRIPTION
**Pull request by AliceAlbano**

### CODE REVIEW

File: end-user/en/source/pages/blueprint/blueprint-deploy.rst [hunk 1]
```diff
@@ -17,8 +17,10 @@ To deploy a blueprint:
 	#. Select the cloud account you want to use to deploy the blueprint. Currently only AWS is supported.
 
 	#. From the drop-down menu, select the ``Region`` in which the blueprint should be deployed. This should be the same region used to publish the appliances that are included in the blueprint.
+	#. On the bottom of the page, there is a list of the appliances used by your blueprint. For each of these appliances, you can see their current status. All the appliances need to be ready.
```
Comment (Numerinico) [inline end-user/en/source/pages/blueprint/blueprint-deploy.rst]: maybe add a comment explaining what "ready" mean ?
Something like. By ready we mean that an appliance in a given revision is published to the selected region.

Comment (AliceAlbano) [reply end-user/en/source/pages/blueprint/blueprint-deploy.rst]: done :)

File: end-user/en/source/pages/blueprint/blueprint-deploy.rst [hunk 2]
```diff
@@ -17,8 +17,10 @@ To deploy a blueprint:
 	#. Select the cloud account you want to use to deploy the blueprint. Currently only AWS is supported.
 
 	#. From the drop-down menu, select the ``Region`` in which the blueprint should be deployed. This should be the same region used to publish the appliances that are included in the blueprint.
+	#. On the bottom of the page, there is a list of the appliances used by your blueprint. For each of these appliances, you can see their current status. All the appliances need to be ready (an appliance is ready when its last version is published to the selected region).
```
Comment (Numerinico) [inline end-user/en/source/pages/blueprint/blueprint-deploy.rst]: revision in UForge ? could be confusing.
or in "it latest version"
maybe @emuus know what is the best

Comment (emuus) [reply end-user/en/source/pages/blueprint/blueprint-deploy.rst]: @Numerinico  its last version -> is ok for me.

Comment (AliceAlbano) [reply end-user/en/source/pages/blueprint/blueprint-deploy.rst]: @Numerinico do you agree?

Comment (Numerinico) [reply end-user/en/source/pages/blueprint/blueprint-deploy.rst]: @emuus is the master :D agree

**Pull request merged**